<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ep_register</class>
 <widget class="QDialog" name="ep_register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register form</string>
  </property>
  <widget class="QGroupBox" name="groupBoxRegister">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Register</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelRegisterUsername">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="RegisterUsername"/>
    </item>
    <item>
     <widget class="QLabel" name="labelRegisterPassword">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="RegisterPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelRegisterRepeat">
      <property name="text">
       <string>Repeat password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="RegisterRepeat">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelRegisterEmail">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="RegisterEmail"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonRegister">
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
